management/virtualmachine/resourceextensions_test.go

```diff
@@ -0,0 +1,38 @@
+package virtualmachine
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management/testutils"
+)
+
+func init() {
+	flag.BoolVar(&extraVerbose, "resourceextensions.verbose", false, "List all resource extensions")

```

Comment:
No need for a lecture, I know all that. Likewise, if there was an extra level of verbosity, people would use it. I had fun figuring out how to add a flag, so I'm good. :-) Code is gone and this diff is outdated.